modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-consume.adoc

```diff
@@ -0,0 +1,93 @@
+= MQ Consume Operation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The `consume` operation in the Anypoint MQ connector provides the ability to consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+
+=== Incoming Message Structure
+
+Every consumed message consists of two parts:
+
+* `payload` - Represents the value of the received message body
+* `attributes` - Contains metadata regarding the received message, such as the message properties, or its ID.
+
+image::anypoint-mq/3.x/amq-3x-consume-attributes.png[Anypoint MQ Message Attributes]
+
+
+== How to Consume a Message
+
+The simplest declaration to consume a message from a queue is to drag the `Consume` operation from the palette to the canvas and configure the `Queue` parameter with the name of the queue from which to retrieve the message:
+
+image::anypoint-mq/3.x/amq-3x-consume-operation.png[Anypoint MQ Consume Operation]
+
+For example:
+
+[source,console,linenums]
+----
+<anypoint-mq:consume config-ref="Anypoint_MQ_config" destination="myQueue"/>
+----
+
+The example operation consumes the first available message in the queue identified by the `Queue` called `myQueue`, converting it to a `MuleMessage` with the following structure:
+
+* The message's body as payload.
+* The message's metadata in the message attributes. Attributes might include the `messageId` or the `redeliveryCount`.
+
+TIP: Use a Dataweave expression in the `Queue` parameter to dynamically change the target queue from which to consume the message.
+
+
+== Consumed Message Acknowledgment
+
+* *Automatic*
++
+By default, the `acknowledgementMode` used is `IMMEDIATE`, meaning that the message is acknowledged as soon as it is consumed from the queue and before it's delivered to the Mule flow. If the message acknowledgment fails, the `Consume` operation terminates with an `ANYPOINT-MQ:ACKING` error.
++
+For more information, see xref:anypoint-mq/3.x/anypoint-mq-ack.adoc#immediate-acknowledgment[Immediate Acknowledgment].
+
+* *Manual*
++
+To manually control the acknowledgment of the message, change the `acknowledgementMode` to `MANUAL`. When using the `MANUAL` acknowledgment mode, the application logic decides when to perform the acknowledgment of the message, using the xref:anypoint-mq/3.x/anypoint-mq-ack.adoc[ACK] or xref:anypoint-mq/3.x/anypoint-mq-ack.adoc[NACK] operations.
++
+To perform the manual acknowledgment, save the value of the `ackToken` provided as part of the resulting message attributes.
++
+For more information, see xref:anypoint-mq/3.x/anypoint-mq-ack.adoc#manual-acknowledgment[Manual Acknowledgment].
+
+For information on acknowledgment timeouts, see xref:anypoint-mq/3.x/anypoint-mq-ack.adoc#acknowledgment-timeout[Acknowledgment Timeout].
+
+== Dealing with Empty Queues
```

Comment:
"Dealing with" is jargon that will be harder to translate. Better is "Handle Empty Queues"